app/models/user.rb

```diff
@@ -1,21 +1,20 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  include Gravtastic
+
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable, :rememberable and :omniauthable
   devise :database_authenticatable, :registerable, :recoverable, :validatable
 
+
   has_many :access_tokens,
            class_name: 'Doorkeeper::AccessToken',
            foreign_key: :resource_owner_id,
            inverse_of: :resource_owner,
            dependent: :delete_all
 
-  delegate :avatar_url, to: :user_decorator
-
-  private
+  is_gravtastic
 
-  def user_decorator
-    @user_decorator ||= UserDecorator.new(self)
-  end
+  alias :avatar_url :gravatar_url
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/6cf75b453495421fff9f79f202ed6993145f73fe/app/models/user.rb#L19"></span>Use <code>alias avatar_url gravatar_url</code> instead of <code>alias :avatar_url :gravatar_url</code>.</td>
    </tr>
  </tbody>
</table>